audio/soundfont/vgmcoll.cpp

```diff
@@ -185,7 +185,7 @@ SynthFile *VGMColl::CreateSynthFile(VGMInstrSet *theInstrSet) {
 				newRgn->SetWaveLinkInfo(0, 0, 1, (uint32) realSampNum);
 
 				if (realSampNum >= finalSamps.size()) {
-					debug("Sample %lu does not exist", realSampNum);
+					debug("Sample %u does not exist", unsigned(realSampNum));
```

Comment:
```suggestion
					debug("Sample %u does not exist", (uint32) realSampNum);
```

Reply:
I agree that `uint32` is more portable than `unsigned`, but why do you prefer C-style cast over ctor-like cast, which is usually a better choice for C++?

Reply:
In this case, mostly for portability reasons, as well as uniformity with the rest of the code